src/components/Charts/Charts.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Doughnut } from 'react-chartjs-2';
+
+export default function Charts(props) {
```

Comment:
Let's re-use the wording `Doughnut` here since we can not fully confirm that this type will the only chart.

Reply:
@glennreyes Thank you for your suggestion. I will rework it a bit. Greetings!

Reply:
@glennreyes I found a way to do this, however, this will import everything from the chartjs library and will increase the js file size. What do you think about this: leaving it as it is, if a new chart type is needed, a switch case or a new component will be added.

Thank you for your time

Reply:
Updated.